README.md

```diff
@@ -5,11 +5,11 @@
 [![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com)
 
 ## Documentation
-This repository is a part of the ShotGrid Pipeline Toolkit.
+This repository is a part of the Flow Production Tracking Toolkit Pipeline Toolkit.
 
 - For more information about this app and for release notes, *see the wiki section*.
 - For general information and documentation, click here: https://help.autodesk.com/view/SGDEV/ENU/?contextId=SA_INTEGRATIONS_USER_GUIDE
-- For information about ShotGrid in general, click here: https://www.autodesk.com/products/shotgrid/overview
+- For information about Flow Production Tracking Toolkit in general, click here: https://www.autodesk.com/products/shotgrid/overview
```

Comment:
I don't think this refers to Toolkit but to SG/Flow. So I think this (and similar ones below) should be:

```patch
- ShotGrid
+ Flow Production Tracking
```